tests/tensorflow/nightly/perfzero.libsonnet

```diff
@@ -0,0 +1,275 @@
+local common = import "common.libsonnet";
+local timeouts = import "templates/timeouts.libsonnet";
+local tpus = import "templates/tpus.libsonnet";
+local gpus = import "templates/gpus.libsonnet";
+local utils = import "templates/utils.libsonnet";
+
+{
+  local PerfZeroTest = common.ModelGardenTest {
+    local config = self,
+
+    timeout: timeouts.one_hour,
+    schedule: "0 8 * * *",
+    mode: "perfzero",
+
+    benchmarkOptions:: {
+      test: {
+        module: null,
+        class: null,
+        method: null,
+      },
+      table: {
+        project: "xl-ml-test",
+        name: "perfzero_dataset.perfzero_table",
+      },
+    },
+    command: [
+      "python3",
+      "/benchmarks/perfzero/lib/benchmark.py",
+      "--gcloud_key_file=",
```

Comment:
making sure this is intended and not that the arg was redacted by Github or something

Reply:
Yes, this need to be an empty string to indicate that we should use the environment ADC instead of a key file.

Reply:
So just to clarify, this means that this runner doesn't require the gcloud key hack that the MLCompass runner does?

Reply:
That's correct. The test runs in GKE, which provides application default credentials instead: https://cloud.google.com/docs/authentication/production